python3/httplib2/__init__.py

```diff
@@ -125,7 +146,7 @@ class CertificateValidationUnsupportedInPython31(HttpLib2Error): pass
 HOP_BY_HOP = ['connection', 'keep-alive', 'proxy-authenticate', 'proxy-authorization', 'te', 'trailers', 'transfer-encoding', 'upgrade']
 
 # Default CA certificates file bundled with httplib2.
-CA_CERTS = os.path.join(
+DEFAULT_CA_CERTS = os.path.join(
```

Comment:
Unfortunately, this name is used by some projects as part of our API, so there has to be a good reason for change.

Reply:
Just wanted to make the code cleaner but you are right about the breaking change. Will revert.